elements/pfe-styles/src/pfe-typography.scss

```diff
@@ -0,0 +1,65 @@
+@import "../../pfe-sass/pfe-sass";
+
+// HEADINGS
+.pfe-heading-3xl {
+    @extend %pfe-heading--3xl;
+}
+.pfe-heading-3xl {
+    @extend %pfe-heading--3xl;
+}
+.pfe-heading-2xl {
+    @extend %pfe-heading--2xl;
+}
+.pfe-heading-xl {
+    @extend %pfe-heading--xl;
+}
+h1,
+.pfe-heading-lg {
+    @extend %pfe-heading--lg;
+}
+h2,
+.pfe-heading-md {
+    @extend %pfe-heading--md;
+}
+h3,
+.pfe-heading-sm {
+    @extend %pfe-heading--sm;
+}
+h4, h5, h6,
+.pfe-heading-xs {
+    @extend %pfe-heading--xs;
+}
+h1, h2, h3, h4, h5, h6,
+[class^="pfe-heading"]:not(:last-child) {
+    margin-bottom: pfe-var(content-spacer--heading--sm); //16px
+}
+.pfe-heading-xxl:not(:last-child),
+.pfe-heading-xl:not(:last-child) {
+    margin-bottom: pfe-var(content-spacer--heading--lg); //32
+}
+ 
+// TEXT
+.pfe-text-lg {
+    @extend %pfe-text-lg;
+}
+p,
+.pfe-text {
+    @extend %pfe-text;
+}
+.pfe-text-sm {
+    @extend %pfe-text-sm;
+}
+.pfe-text-xs {
+    @extend %pfe-text-xs;
+}   
+
+//  a {
```

Comment:
Do we want to move link styles out of theme and into PFE?

Reply:
I think allowing people to opt-in as much as possible is ideal. I'm going to split these out for now, we can always re-combine them if needed.

Reply:
the link styles now exist in the pfe-text-elements.css